manuscript/frontmatter.md

```diff
@@ -12,28 +12,28 @@ I also would like to thank Trishul, Santhosh, and Karthic the rockstar contribut
+The first two chapters **Getting Started with the WebExtensions API** and **Running Extensions in Firefox** will help you learn about the basics of browser extensions and teach you how to configure your browser for developing the extension and how to run them locally.
 
-- In *Experiment 1*, you will be learning how to create very basic simple extension without much need of Javascript. If you are learning JS for the first time, even then you can work on this experiment.
+- In *Experiment 1*, you will learn how to create very basic simple extension without much need of Javascript. If you are learning JS for the first time, you can work on this experiment.
 
-- In *Experiment 2*, we will be getting started with the first user interface (UI) with related to Browser Extension. We will be getting the search engines in our browser and create a context menu and do the interaction with them. Context menus are most used UI in the browser by average users.
+- In *Experiment 2*, we will be getting started with the first user interface (UI) with related to browser extensions. We will be getthe search engines in our browser and create a context menu and do the interaction with them. Context menus are most used UI in the browser by average users.
 
-- In *Experiment 3*, we will be learning about the second type of user interface of Browser Extension.  We will be learning about browser Action icon, pop-up HTML page, and interaction between the page and background script.
+- In *Experiment 3*, we will learn about the second type of user interface for extensions.  We will learn about browserAction, pop-up HTML pages, and interactions between the page and background script.
 
-- In *Experiment 4*, we will be learning about the third type of user interface of Browser Extension.  We will be learning about page Action icon and its interaction between the page and background script. This is similar to browserAction but with minimal difference.
+- In *Experiment 4*, we will learn about the third type of user interface for xtensions.  We will learn about page actions and their interactions between the page and background script. This is similar to browserAction but with minimal difference.
 
-- In *Experiment 5*, we will be learning about the notifications in BrowserExtension.   This is one of the most interesting experiment and is my favorite one. This is very easy and most enjoyable experiment among all.
+- In *Experiment 5*, we will be learning about notifications in extensions. This is one of the most interesting experiment and is my favorite one. This is very easy and most enjoyable experiment among all.
 
 - In *Experiment 6*, we will be learning about the shortcuts in browser interacting via BrowserExtension.   All the experiment before this we used to have some sort of things which can be displayed, but in this experiment, we will be doing everything in the background.
```

Comment:
I didn't understand your summary of Experiment 6 -- you might want to clean this up.